trento-docs-site/antora-playbook.yml

```diff
@@ -28,4 +28,21 @@ antora:
 asciidoc:
   attributes:
     page-pagination: ''
-    site-gen-antora: ''
\ No newline at end of file
+    site-gen-antora: ''
+    imagesdir: _images
+    script_name: 'generate-components.nav.js'
```

Comment:
These variables are set for now to be used in documents for maintenance, i'm considering to move the docs component outside in order to not have a big list of variables. These variables apply only to content directory!